packages/wallets/src/components/ErrorState/ErrorState.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import IcCashierError from '../../public/images/ic-cashier-error.svg';
+import FundsProtection from '../FundsProtection';
+import { GenericPageWithAction } from '../GenericPageWithAction';
+
+type TProps = {
+    error: unknown;
+};
+
+const ErrorState: React.FC<TProps> = ({ error }) => {
+    let code = 'UNKNOWN';
+    let message = 'Sorry an error occured. Please try accessing our cashier page again.';
+
+    if (error && typeof error === 'object' && 'code' in error && typeof error.code === 'string') {
+        code = error.code;
```

Comment:
for the error code - is it necessary? typically its not beneficial for clients to see technical details / code errors etc